phpBB/phpbb/feed/base.php

```diff
@@ -81,8 +84,19 @@
 	* @param \phpbb\auth\auth					$auth		Auth object
 	* @param \phpbb\content_visibility			$content_visibility		Auth object
 	* @param string								$phpEx		php file extension
+	* @param \phpbb\event\dispatcher_interface	$phpbb_dispatcher		Event dispatcher object
 	*/
-	function __construct(\phpbb\feed\helper $helper, \phpbb\config\config $config, \phpbb\db\driver\driver_interface $db, \phpbb\cache\driver\driver_interface $cache, \phpbb\user $user, \phpbb\auth\auth $auth, \phpbb\content_visibility $content_visibility, $phpEx)
+	function __construct(
```

Comment:
In general, constructor arguments should be classes first and then strings and optional parts at the end. Please move the dispatcher in front of the `$phpEx`